spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -917,10 +938,13 @@
             find("div#bulk-actions-dropdown div.menu_item", text: "Delete Selected" ).click
 
             within ".modal" do
-              expect(page).to have_content "This will delete one line item from the order. Are you sure you want to proceed?"
+              expect(page).to have_content "This will delete one line item from the order. " \
+                                           "Are you sure you want to proceed?"
               click_on "OK"
             end
 
+            expect(page).to have_content "Loading orders"
+
```

Comment:
I think that this was removed intentionally because it was leading to flaky specs. You accidentally added it when resolving merge conflicts.